tests/integration/test_statement_execution.py

```diff
@@ -146,7 +146,8 @@ def execute_statement_and_return_index_value(txn, query):
             # {
             #    expr: "[MyColumn]"
             # }
-            return next(cursor)['expr']
+            for row in cursor:
+                return row['expr']
```

Comment:
Return the cursor and read it outside of the transaction. This can fail by not returning anything.

Reply:
Updated